frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh

```diff
@@ -1,37 +1,17 @@
-KUBEVIRT_VERSION=$(curl -s https://github.com/kubevirt/kubevirt/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
-#CDI_VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
-CDI_VERSION="v1.29.0"
+#!/bin/bash
 
-VIRTCTL_DOWNLOAD_URL="https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/virtctl-${KUBEVIRT_VERSION}"
-VIRTCTL_X86_64="${VIRTCTL_DOWNLOAD_URL}-linux-x86_64"
-VIRTCTL_AMD64="${VIRTCTL_DOWNLOAD_URL}-linux-amd64"
+HCO_VERSION="master"
+
+curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/$HCO_VERSION/deploy/deploy.sh | bash
 
 # Create openshift-cnv namespace for Integration Tests
 oc create namespace openshift-cnv
 
-oc create namespace kubevirt-os-images
-
-# Deploy Kubevirt, Storage, CDI Pods
-oc create -f https://github.com/kubevirt/kubevirt/releases/download/$KUBEVIRT_VERSION/kubevirt-operator.yaml
-
-oc create -f https://github.com/kubevirt/kubevirt/releases/download/$KUBEVIRT_VERSION/kubevirt-cr.yaml
-
-# File was deleted upstream, using last known file in github, until a more permanent fix:
-# https://github.com/openshift/console/pull/8608 - will fix this issue.
-# oc create -f https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/master/labs/manifests/storage-setup.yml
-oc create -f https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/f6530b3fe71e8821208cad8fcac165c54a42bd54/labs/manifests/storage-setup.yml
-
-oc create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$CDI_VERSION/cdi-operator.yaml
-
-oc create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$CDI_VERSION/cdi-cr.yaml
-
-# Deploy Common Templates
-oc project openshift
-oc create -f https://github.com/kubevirt/common-templates/releases/download/v0.13.1/common-templates-v0.13.1.yaml
-oc project default
+# Setup storage
+oc create -f ./ci-scripts/hostpath-provisioner-setup.yml
```

Comment:
this script is going to run from a different server, you need to use `oc create -f http://....`

so lets do a PR that adds the `ci-scripts` files, merge them, and then continue with this PR ...

Reply:
I created https://github.com/openshift/console/pull/8654 to add the ci-scripts files and removed them from the script in this PR.